integration/auth/user_mgt_test.go

```diff
@@ -249,9 +249,22 @@ func TestLastRefreshTime(t *testing.T) {
 		t.Errorf("signInWithPassword failed: %v", err)
 	}
 
-	getUsersResult, err := client.GetUser(context.Background(), userRecord.UID)
-	if err != nil {
-		t.Fatalf("GetUser(...) failed with error: %v", err)
+	// Attempt to retrieve the user 3 times (with a small delay between each attempt.) Occasionally,
+	// this call retrieves the user data without the lastLoginTime/lastRefreshTime fields; possibly
+	// because it's hitting a different server than what the login request used.
+	var getUsersResult *auth.UserRecord
+	for i := 0; i < 3; i++ {
+		var err error
+		getUsersResult, err = client.GetUser(context.Background(), userRecord.UID)
```

Comment:
nit: Could you use `:=` here and eliminate the var err statement on the previous line?

Optional.

Reply:
That also declares a new `getUsersResult` variable in the nested scope. We want to reuse the one defined outside the for loop.